templates/Guides/v1_v2_migration.md

```diff
@@ -0,0 +1,22 @@
+---
+page_title: "v1 to v2 Migration - terraform-provider-basistheory"
+subcategory: ""
+description: |-
+  How to migrate from v1 to v2 of the Basis Theory Terraform provider
+---
+
+# v1 to v2 Migration
+
+This is a guide on how to upgrade from v1 to v2 of the Basis Theory Terraform provider. We try to automate as many steps
+as possible for you, but there are a few manual steps that must be done on your end to ensure the upgrade is successful.
+Before upgrading follow the steps for each resource in this guide.
+
+## basistheory_reactor
+
+If you currently have a `basistheory_reactor` with a `formula_id`, you will need to create a new reactor the `code` that
```

Comment:
I think I commented above on the file produced by this template? We should fix that typo here too, but also is it expected for this file to be checked in twice?

Reply:
the migration doc under the `docs` folder is generated from this template. I had to add it here so that our docs check passes. we check that docs are updated, most of them are autogenerated though so this is a new `template` dir with this `md` file